### PR TITLE
Add failing unit tests

### PR DESCRIPTION
**Pull request by JSteunou**
linked to #4103

### CODE REVIEW

File: test/unit/model/bulkcreate.test.js [hunk 1]
```diff
@@ -45,5 +65,19 @@ describe(Support.getTestDialectTeaser('Model'), function() {
...
       });
     });
+
+    describe('renamed fields', function () {
+      it('should not fail for timestamps', function () {
+        return ModelWithTS.bulkCreate([
+          { accountId: 42 }
+        ]).bind(this).then(function () {
+          var firstCreatedModel = this.stub.getCall(0).args[1][0];
+          expect(firstCreatedModel).to.have.property('id');
```
Comment (mickhansen) [inline test/unit/model/bulkcreate.test.js]: `id` would not ever be present in the bulkInsert call.

Comment (JSteunou) [inline test/unit/model/bulkcreate.test.js]: I add it because it's present (but null) so I though it was normal

Comment (mickhansen) [inline test/unit/model/bulkcreate.test.js]: Ah, that might be true, but it's not something that's relevant to this test :)

**Comment by JSteunou** [comment]
Updated to remove check on `id`

**Pull request closed without merging**